sentry-android-core/src/main/java/io/sentry/android/core/SentryLogcatAdapter.java

```diff
@@ -44,73 +48,108 @@ private static void addAsBreadcrumb(
     Sentry.addBreadcrumb(breadcrumb);
   }
 
+  private static void addAsLog(
+      @NotNull final SentryLogLevel level,
+      @Nullable final String msg,
+      @Nullable final Throwable tr) {
+    final @NotNull ScopesAdapter scopes = ScopesAdapter.getInstance();
+    // Check if logs are enabled before doing expensive operations
+    if (!scopes.getOptions().getLogs().isEnabled()) {
+      return;
+    }
+    if (tr == null) {
+      scopes.logger().log(level, msg);
+    } else {
+      StringWriter sw = new StringWriter(256);
+      PrintWriter pw = new PrintWriter(sw, false);
+      tr.printStackTrace(pw);
```

Comment:
Are you sure we want to add the whole stacktrace to logs as opposed to asking customers to use issues for them? Maybe confirm internally (#proj-logs)